### PR TITLE
Additional Clothing

### PR DESCRIPTION
**Pull request by Esdian**
Adding in a few extra articles of clothing, thigh high boots and bikini bottoms.  I really do need some some more options on the recipe for the bikini bottom, as well as placement in item groups for everything.  Also planing on adding in some long white gloves, but I still need to work on those.

Added gloves, tho still need to put them in item groups and such.

### CODE REVIEW

**Comment by chaosvolt** [comment]
I am sorely tempted to suggest a single-sling backpack. owo

**Comment by Esdian** [comment]
That is  something I'd not thought of.  Might have to fiddle around with making one, but I'm afraid it would just come out as a re-skined messenger bag. 

I do need some help setting up the item groups, as I'm not sure where the gloves would fit. Also The exact rates on the bikini bottoms showing up seems off to me, but that’s just me.

**Comment by chaosvolt** [comment]
Well, from what I've seen of most sling packs, they'd likely be in-between messenger bags and backpacks in terms of both encumbrance and storage. On the one hand they're easier to shift about so they don't catch on stuff compared to a normal pack, but I've also found them less comfortable if overloaded. XP

And...hmm. Since I'm guessing they're effectively evening gloves, any item group set aside for fancy clothing would be a logical place, though they could also be in allclothes and other item groups of that nature, likely at a much lower rate.

As for bikini bottoms, unsure. Depends on if what other items are already in that group. Would presumably be of comparable rarity to whatever the equivalent presumed-male garment in the itemgroup would be, but not sure how to weight them relative to other items in the group.

File: data/json/items/armor.json [hunk 1]
```diff
@@ -13025,5 +13025,79 @@
             "watertight" : true,
             "seals" : true
         }
-   }
+   },    
+   {
+        "type" : "ARMOR",
```
Comment (kevingranade) [inline data/json/items/armor.json]: Can you insert these near similar items instead of at the end of the file?

**Comment by Esdian** [comment]
Best I can tell, closest Alternative for bottoms would be Hot pants, If I'm reading the item_group json right. Maybe, Just toss in the Bikini bottoms at a similar rate to hot pants? Currently for testing, I've added them to almost every place "trunks" are at a similar rate. (note not all changes are reflected in the PR yet)

For the sling pack, encumbrance to carry, maybe something like 8 encumbrance to carry 25 or 30 volume?

**Comment by Esdian** [comment]
Ok, Tried to re-configure where stuff was. Logic behind locations,Thigh-High boots I placed right after the knee-high boots, Bikini bottom, was placed with Trunks, and its recipe was placed behind panties. The gloves were moved to be in with some other gloves. (lazy search of gloves and placed its info near there)

Let me know how much more I need to work on.

**Comment by Esdian** [comment]
Ok, I think I'm finished now, all that left is you guys reviewing everything.  For a quick summary of what this includes, 4 new items, Bikini bottom, thigh high boots, extra long gloves, and an evening gown.  The Bikini bottom was missing in general,  I mean we can make Bikini tops, and panties so felt like the logical extension.  The boots, literally just my personal taste, and thought it would be a neat addition. The gloves I came up with while thinking about the French maid start, seems like it would fit in that whole look as well as generally being nice. And lastly the Evening Gown is an alternative to the tuxedo.

File: data/json/items/armor.json [hunk 2]
```diff
@@ -2883,6 +2932,30 @@
...
         "type" : "ARMOR",
+        "id" : "gown",
+        "name" : "evening gown",
+        "weight" : 1300,
+        "color" : "black",
+        "covers" : ["TORSO", "LEGS"],
+        "to_hit" : 0,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "A luxurious evening gown. It accentuates your curves, or lack thereof.",
```
Comment (Rivet-the-Zombie) [inline data/json/items/armor.json]: Double spacing after periods, please.

**Comment by chaosvolt** [comment]
Ah right. Other than the obligatory double spacing, looks okay.

**Comment by Rivet-the-Zombie** [comment]
Yeah other than that, it looks fine.

**Comment by Esdian** [comment]
Checked, made the change.

**Comment by illi-kun** [comment]
PR looks fine, the only thing I worried about is weight of the evening gown (1.3 kg).
Quick googling provides me [these results for a regular dress] (http://www.cockeyed.com/science/weight/dress-womans.html), so from my point of view you need to cut down the current value.

**Comment by Esdian** [comment]
After some research, It seems the gown might be under weight.  Using http://www.amazon.com/Beaded-Mesh-Fairy-Dress-Formal/dp/B00316G2DE as a refrence at least.  Or it could be close if part of the weight listed is shiping related.  2.87 pounds in game *for me at least* vs 4.2 pounds.

**Comment by illi-kun** [comment]
Fair enough, thanks for reference.

**Comment by Esdian** [comment]
Settled on adding about a pound to the weight, I'm happy, with it.

**Pull request merged**